src/main/java/net/consensys/wittgenstein/core/ProgressPerTime.java

```diff
@@ -76,6 +76,7 @@ public void run(Predicate<Protocol> contIf) {
         if (p.network().time % 10000 == 0) {
           System.out.println("time goes by... time=" + (p.network().time / 1000) + ", stats=" + s);
         }
+        liveNodes = p.network().allNodes.stream().filter(n -> !n.down).collect(Collectors.toList());
       } while (contIf.test(p));
```

Comment:
WOuldn't be better to recalculate the liveNodes just after the 'p.network().runMs(10);' above?

Reply:
moved